include/c/xamarin/sk_x_typeface.h

```diff
@@ -23,6 +23,13 @@ SK_API sk_typeface_t* sk_typeface_create_from_file(const char* path, int index);
 SK_API sk_typeface_t* sk_typeface_create_from_stream(sk_stream_asset_t* stream, int index);
 SK_API int sk_typeface_chars_to_glyphs(sk_typeface_t* typeface, const char *chars, sk_encoding_t encoding, uint16_t glyphs[], int glyphCount);
 
+SK_API void sk_typeface_get_family_name(sk_typeface_t* typeface, sk_string_t* family_name);
```

Comment:
@agallero, I think we should change this to a RETURN function as we are returning an `IntPtr`. That way, we can remove the need to create the `SKString` in managed code and reduce the stuff we pass back and forth across the boundaries.